src/screens/ProfileScreen/ProfileScreen.tsx

```diff
@@ -40,39 +33,6 @@ const ProfileScreen = () => {
     return true;
   };
 
-  const fetchUserInfo = async () => {
-    try {
-      const githubLoginInfo = await Keychain.getGenericPassword();
-
-      if (!githubLoginInfo || !githubLoginInfo.password) {
-        return;
-      }
-
-      const data = JSON.parse(githubLoginInfo.password);
-
-      const { accessToken } = data;
-
-      const userAgent = await UserAgent.getWebViewUserAgent();
-
-      axios.defaults.headers.common['User-Agent'] = userAgent;
-
-      const res = await axios.get('https://api.github.com/user', {
-        headers: {
-          'User-Agent': `${userAgent}`,
-          Authorization: `token ${accessToken}`,
-        },
-      });
-
-      console.log('data', JSON.stringify(res.data));
-    } catch (error) {
-      console.log('error', error);
```

Comment:
instead of consoling these errors can we use a any sentry service to track these catch blocks?